docs/source/plugin.rst

```diff
@@ -5,7 +5,12 @@ A Sopel plugin consists of a Python module containing one or more
 ``callable``\s. It may optionally also contain ``configure``, ``setup``, and
 ``shutdown`` hooks.
 
-.. py:method:: callable(bot, trigger)
+.. py:function:: callable(bot, trigger)
+
+    :param bot: the bot's instance
+    :type bot: :class:`sopel.bot.Sopel`
```

Comment:
This is not true. As I discovered earlier, commands are called with an instance of `SopelWrapper`, and not `Sopel`. But shince `SopelWrapper` is not yet available from outside `Sopel`... it's hard to link it in the documentation.

This need to be fixed once this change is done.

Reply:
Ok, so now it needs #1542 to be merged so I can change that.